QOpenGLFunctions_2_0 doesn't exist if Qt was built with opengles enabled.

Ref: https://github.com/KDAB/GammaRay/issues/829

diff --git a/plugins/quickinspector/textureextension/qsgtexturegrabber.cpp b/plugins/quickinspector/textureextension/qsgtexturegrabber.cpp
index 120bb400d..f8d187ceb 100644
--- a/plugins/quickinspector/textureextension/qsgtexturegrabber.cpp
+++ b/plugins/quickinspector/textureextension/qsgtexturegrabber.cpp
@@ -17,7 +17,9 @@
 #include <QImage>
 #include <QOpenGLContext>
 #include <QOpenGLFunctions>
+#if !QT_CONFIG(opengles2)
 #include <QOpenGLFunctions_2_0>
+#endif
 #include <QPainter>
 #include <QQuickWindow>
 #include <QSGTexture>
